frontend/src/locales/scripts/en.json5

```
@@ -1153,9 +1153,9 @@
       },
       source: {
         audio: {
-          zero: "No audio files provided by this source",
-          count: "{count} audio file provided by this source|{count} audio files provided by this source",
-          countMore: "Over {count} audio files provided by this source",
+          zero: "No audio track files provided by this source",
+          count: "{count} audio track file provided by this source|{count} audio track files provided by this source",
+          countMore: "Over {count} audio track files provided by this source",
```

The word "track" and "file" should not both be used as one removes the need for the other.

```suggestion
          zero: "No audio tracks provided by this source",
          count: "{count} audio track provided by this source|{count} audio tracks provided by this source",
          countMore: "Over {count} audio tracks provided by this source",
```